content/tutorials/spatial_standards_vector/index.md

```diff
@@ -343,27 +343,26 @@ habitatstreams_test %>%
 ```
 
     ## Simple feature collection with 560 features and 2 fields
-    ## geometry type:  LINESTRING
-    ## dimension:      XY
-    ## bbox:           xmin: 33013.71 ymin: 157590.5 xmax: 253945.9 ymax: 243502.9
-    ## epsg (SRID):    31370
-    ## proj4string:    +proj=lcc +lat_1=51.16666723333333 +lat_2=49.8333339 +lat_0=90 +lon_0=4.367486666666666 +x_0=150000.013 +y_0=5400088.438 +ellps=intl +towgs84=-106.8686,52.2978,-103.7239,0.3366,-0.457,1.8422,-1.2747 +units=m +no_defs
+    ## Geometry type: LINESTRING
+    ## Dimension:     XY
+    ## Bounding box:  xmin: 33097.92 ymin: 157529.6 xmax: 254039 ymax: 243444.6
+    ## Projected CRS: BD72 / Belgian Lambert 72
     ## First 10 features:
     ##        river_name   source                       geometry
-    ## 1     WOLFPUTBEEK      VMM LINESTRING (127768.8 167742...
-    ## 2       OUDE KALE      VMM LINESTRING (95650.24 196973...
-    ## 3         VENLOOP   EcoInv LINESTRING (169263.1 209374...
-    ## 4         VENLOOP   EcoInv LINESTRING (169544 209352.8...
-    ## 5     KLEINE NETE   EcoInv LINESTRING (180997 208666.5...
-    ## 6     KLEINE NETE   EcoInv LINESTRING (179947.3 208419...
-    ## 7     KLEINE NETE   EcoInv LINESTRING (180429.9 208655...
-    ## 8     KLEINE NETE   EcoInv LINESTRING (187289.6 210058...
-    ## 9  RAAMDONKSEBEEK extrapol LINESTRING (183455.2 192468...
-    ## 10    KLEINE NETE   EcoInv LINESTRING (183426.2 208321...
-
-1.  DOI = Digital Object Identifier. See <https://www.doi.org>.
-
-2.   Though GeoJSON 2008 is
+    ## 1     WOLFPUTBEEK      VMM LINESTRING (127857.1 167681...
+    ## 2       OUDE KALE      VMM LINESTRING (95737.01 196912...
+    ## 3         VENLOOP   EcoInv LINESTRING (169352.7 209314...
+    ## 4         VENLOOP   EcoInv LINESTRING (169633.6 209293...
+    ## 5     KLEINE NETE   EcoInv LINESTRING (181087.1 208607...
+    ## 6     KLEINE NETE   EcoInv LINESTRING (180037.4 208360...
+    ## 7     KLEINE NETE   EcoInv LINESTRING (180519.9 208595...
+    ## 8     KLEINE NETE   EcoInv LINESTRING (187379.9 209998...
+    ## 9  RAAMDONKSEBEEK extrapol LINESTRING (183545.6 192409...
+    ## 10    KLEINE NETE   EcoInv LINESTRING (183516.4 208261...
```

Comment:
This has changed too. I think the previous version was incorrect because it seems that now we are correctly roundtripping when going from BD72 / Belgian Lambert 72 to WGS84 and back. So probably something got fixed in PROJ.

Reply:
Well, it was and will never be 'correct' in the sense that _transformations_, which implicate a datum shift, cause accuracy loss because of an inherently inaccurate relationship between geographical datums (as opposed to _conversions_, e.g. between ETRS89 and Belgian Lambert 2008: they are perfect by definition).

But indeed you've noted this well (probably I didn't), no roundtripping occurred in the previous state, so that means that previously the operation for backtransformation was not the inverse of the forward transformation. There are often multiple alternative pipelines available to perform a transformation, and formulas can evolve, so clearly the defaults were different back then. Currently, the default back- and forward-transformations are each others inverse (cf. description below), so that we get the exact original state back. Indeed, much better.

```r
> sf::sf_proj_pipelines("EPSG:31370", "EPSG:4326")
Candidate coordinate operations found:  4 
Strict containment:     FALSE 
Axis order auth compl:  FALSE 
Source:  EPSG:31370 
Target:  EPSG:4326 
Best instantiable operation has accuracy: 1 m
Description: Inverse of Belgian Lambert 72 + BD72 to WGS 84 (3) + axis order
             change (2D)
Definition:  +proj=pipeline +step +inv +proj=lcc +lat_0=90
             +lon_0=4.36748666666667 +lat_1=51.1666672333333
             +lat_2=49.8333339 +x_0=150000.013 +y_0=5400088.438
             +ellps=intl +step +proj=push +v_3 +step +proj=cart
             +ellps=intl +step +proj=helmert +x=-106.8686
             +y=52.2978 +z=-103.7239 +rx=-0.3366 +ry=0.457
             +rz=-1.8422 +s=-1.2747
             +convention=coordinate_frame +step +inv +proj=cart
             +ellps=WGS84 +step +proj=pop +v_3 +step
             +proj=unitconvert +xy_in=rad +xy_out=deg

> sf::sf_proj_pipelines("EPSG:4326", "EPSG:31370")
Candidate coordinate operations found:  4 
Strict containment:     FALSE 
Axis order auth compl:  FALSE 
Source:  EPSG:4326 
Target:  EPSG:31370 
Best instantiable operation has accuracy: 1 m
Description: axis order change (2D) + Inverse of BD72 to WGS 84 (3) +
             Belgian Lambert 72
Definition:  +proj=pipeline +step +proj=unitconvert +xy_in=deg +xy_out=rad
             +step +proj=push +v_3 +step +proj=cart
             +ellps=WGS84 +step +inv +proj=helmert +x=-106.8686
             +y=52.2978 +z=-103.7239 +rx=-0.3366 +ry=0.457
             +rz=-1.8422 +s=-1.2747
             +convention=coordinate_frame +step +inv +proj=cart
             +ellps=intl +step +proj=pop +v_3 +step +proj=lcc
             +lat_0=90 +lon_0=4.36748666666667
             +lat_1=51.1666672333333 +lat_2=49.8333339
             +x_0=150000.013 +y_0=5400088.438 +ellps=intl

```